### ISSUE TITLE
time-series subject audit question 3 of Exercise 4

### ISSUE THREAD

**Issue by jarmo-seljamaa** [01-edu/public#2300]
numpy
---
The [expected sample output for question 3 of Exercise 4](https://github.com/01-edu/public/tree/master/subjects/ai/time-series/audit#for-question-3-is-the-pnl-computed-as-signal--futur_return-both-series-should-have-the-same-index) is identical to the [expected output of question 1](https://github.com/01-edu/public/tree/master/subjects/ai/time-series/audit#for-question-1-is-the-return-computed-as-returnt--pricet1---pricetpricet-and-returns-the-following-output-note-that-if-the-index-is-not-ordered-in-ascending-order-the-future-return-computed-is-wrong-the-answer-is-also-accepted-if-the-returns-is-computed-as-in-the-exercise-2-and-then-shifted-in-the-futur-using-shift-but-i-do-not-recommend-this-implementation-as-it-adds-missing-values). This would be possible only if the random numbers generated in [audit question 2](https://github.com/01-edu/public/tree/master/subjects/ai/time-series/audit#for-question-2-is-the-index-of-the-series-the-same-as-the-index-of-the-dataframe-the-data-of-the-series-can-be-generated-using-nprandomrandint02lendfindex) had five 1 in the beginning and five 1 in the end of the series and no 0 at all. 

In that unlikely event, please regenerate the random array, so the example of question 3 would make more sense to the students.

**Comment by nprimo** [comment]
Hi @jarmo-seljamaa, thank you for the feedback - I will update the expected result for question 3 with the proper values.

(By the way, this is `time-series` not `numpy` project)

**Comment by jarmo-seljamaa** [comment]
My apologies for the wrong reference, but the link to submit an issue on 01 platform becomes active only after the tasks of that day are open (which is an annoying feature of the platform). Like I mentioned [here](https://github.com/01-edu/public/issues/2297#issue-2008753572) I was looking ahead at the next day's tasks and therefore used the link below numpy task to open a new issue. I'm sorry for making a mistake while helping you fix the mistakes. :))
